.travis.yml

```diff
@@ -16,17 +16,18 @@ before_install:
   - export GALAXY_REPO=https://github.com/galaxyproject/galaxy
   - export GALAXY_RELEASE=release_16.04
   - export CONDA_PREFIX=$HOME/conda
+  - export GALAXY_CONFIG_CONDA_DEBUG=True
+  - travis_retry sudo apt-get install samtools
 
 install:
   - pip install flake8
-  - pip install -q click==5.1
   - pip install -q planemo
   - planemo conda_init --conda_prefix $CONDA_PREFIX
-  - export PATH=$CONDA_PREFIX/bin:$PATH
-  - conda install -y -q conda=3.19.3
-  - conda create -y -q -c bioconda --name iuc_conda numpy==1.9.2 samtools=0.1.19 bcftools
-  - source activate iuc_conda
+  - $CONDA_PREFIX/bin/conda install -y -q conda=3.19.3
```

Comment:
This shouldn't be needed any more since Planemo now downgrades to 3.19.3 already. If you look at the Travis logs, this command actually just upgrades some other packages, don't know why. Very confusing!